src/core/exec-invoke.c

```diff
@@ -2244,14 +2244,42 @@ static int build_pass_environment(const ExecContext *c, char ***ret) {
         return 0;
 }
 
-_noreturn_ static void bpffs_helper(int parent_fd) {
+_noreturn_ static void bpffs_helper(int parent_fd,
+                                    uint64_t delegate_cmds,
+                                    uint64_t delegate_maps,
+                                    uint64_t delegate_progs,
+                                    uint64_t delegate_attachs) {
         _cleanup_close_ int fs_fd = -EBADF, mnt_fd = -EBADF;
+        char number[16 + 2 + 1];
```

Comment:
Please make this more descriptive, e.g.
`STRLEN("0x") + sizeof(uint64_t) * 2 + 1` ?